src/controllers/accommodation.controller.js

```diff
@@ -42,6 +45,27 @@ class AccommodationController {
     ResponseService.setSuccess(200, 'list of available accommodations', availableAccommodations);
     return ResponseService.send(res);
   }
+
+  /**
+       *
+       *
+       * @static
+       * @param {req} req
+       * @param {res} res
+       * @returns {response} @memberof AccommodationController
+       */
+  static async likeOrUnlikeAccommodation(req, res) {
```

Comment:
this function is called `likeOrUnlikeAccommodation ` while you are using `like` and `dislike`. Could do you put some similarities?